config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-discovery.yaml

```diff
@@ -0,0 +1,42 @@
+# periodic jobs
+
+periodics:
+- name: ci-kubernetes-e2e-kubeadm-kinder-discovery-master
+  interval: 2h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-kubeadm,sig-release-master-informing
```

Comment:
we should not have these in `sig-release-master-informing`, yet.
needs a stable period and then approval from sig release.

we also don't have the external etcd jobs there, yet.

[1]